man/journald.conf.xml

```diff
@@ -368,11 +368,13 @@
         <term><varname>ForwardToKMsg=</varname></term>
         <term><varname>ForwardToConsole=</varname></term>
         <term><varname>ForwardToWall=</varname></term>
+        <term><varname>ForwardAddress=</varname></term>
```

Comment:
I think this should probably be called ForwardToSocket=?

i mean, address is so generic, could be a memory address or something. But "socket" makes very clear that this is about AF_UNIX, AF_VSOCK and friends.

Reply:
I would prefer if the name indicated that it wasn't a boolean like the others. Would SocketForwardAddress work? If not I'm happy to go with ForwardToSocket.